types/v1/math.go

```diff
@@ -312,3 +312,25 @@ func removeBigIntFactors(d, factor *big.Int) (result *big.Int, times int32) {
 	}
 	return d, times
 }
+
+// SubstractFloat64 returns a - b , where a > b.
+func SubstractFloat64(a float64, b float64) float64 {
+	return (a - b)
+}
+
+// DivideFloat64 returns the value of value divided by base.
+// divide by zero case handled before calling.
```

Comment:
Why is the divide by zero handled somewhere else ?
I feel this function is the right place

Reply:
I handled this case before calling this function. that was requirement. although i will move all these operations to separate functions, these are required for calculating latency. An issue [#856](https://github.com/openebs/openebs/issues/856)